packages/shared/src/components/modals/CreateMyFeedModal.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { ModalProps } from './StyledModal';
+import { ResponsiveModal } from './ResponsiveModal';
+import styles from './AccountDetailsModal.module.css';
+import TagsFilter from '../filters/TagsFilter';
+import CreateFeedFilterButton from '../CreateFeedFilterButton';
+import PlusIcon from '../../../icons/plus.svg';
+import XIcon from '../../../icons/x.svg';
+import { Button, ButtonSize } from '../buttons/Button';
+
+export type TypeProps = {
+  type: string;
+};
+
+export type CreateMyFeedModalProps = TypeProps & ModalProps;
+
+export type LayoutModalProps = TypeProps & Partial<ModalProps>;
```

Comment:
I see you export all these, are they used anywhere else?

Reply:
They are not 👍